From: Andrew Morton <akpm@linux-foundation.org>
Date: Mon, 7 Dec 2020 19:41:57 -0800
Subject: revert "mm/filemap: add static for function __add_to_page_cache_locked"
Patch-mainline: Submitted - 2020-12-07 - http://lkml.kernel.org/r/20201208034157.9KFja%akpm@linux-foundation.org
References: http://lkml.kernel.org/r/20201207081556.pwxmhgdxayzbofpi@lion.mk-sys.cz

Revert 3351b16af494 ("mm/filemap: add static for function
__add_to_page_cache_locked") due to build issues with
CONFIG_DEBUG_INFO_BTF=y.

Link: https://lkml.kernel.org/r/CAADnVQJ6tmzBXvtroBuEH6QA0H+q7yaSKxrVvVxhqr3KBZdEXg@mail.gmail.com
Cc: Michal Kubecek <mkubecek@suse.cz>
Cc: Justin Forbes <jmforbes@linuxtx.org>
Cc: Alex Shi <alex.shi@linux.alibaba.com>
Cc: Souptick Joarder <jrdr.linux@gmail.com>
Cc: Alexei Starovoitov <ast@kernel.org>
Cc: Daniel Borkmann <daniel@iogearbox.net>
Cc: Josef Bacik <josef@toxicpanda.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Acked-by: Michal Kubecek <mkubecek@suse.cz>
---
 mm/filemap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/filemap.c
+++ b/mm/filemap.c
@@ -827,7 +827,7 @@ int replace_page_cache_page(struct page *old, struct page *new, gfp_t gfp_mask)
 }
 EXPORT_SYMBOL_GPL(replace_page_cache_page);
 
-static noinline int __add_to_page_cache_locked(struct page *page,
+noinline int __add_to_page_cache_locked(struct page *page,
 					struct address_space *mapping,
 					pgoff_t offset, gfp_t gfp,
 					void **shadowp)
